config/locales/views/sulten/en.yml

```diff
@@ -83,10 +88,182 @@ en:
       add_new_link: 'Add new period'
       archive_title: 'Previous closed periods for Lyche'
       show_all_previous_closed_periods: 'Show all previous closed periods'
-
       creation_success: "Successfully created Lyche reservations closed period"
       creation_failure: "Failed at creating Lyche reservations closed period"
       update_success: "Successfully updated Lyche reservations closed period"
       update_failure: "Failed at updating Lyche reservations closed period"
       destruction: "Successfully destroyed Lyche reservations closed period"
       times_in_valid_order: "End must be after beginning"
+    lyche:
+      index:
+        reservation:
+          title: "Reservation"
+          description: "Do you want to reserve a table at Lyche? You can do this through our reservation system, or by e-mail."
+          table: "Reserve a table"
+        menu:
+          title: "Menu"
+          description: "At Lyche, we want to emphasize high-quality drinks and food. This applies both to having a varied but also a well-thought-out selection. We make our dishes with a focus on making them from scratch with local ingredients."
+          goto_menu: "See our menu"
+        about:
+          title: "About lyche"
+          description: "Lyche is a restaurant run on a voluntary basis. All the waiters and chefs in the kitchen work voluntarily, and we are part of the Café and serving group (KSG) at the Student Society in Trondheim."
+          goto_about: "More about us"
+        contact:
+          title: "Contact"
+          description: "Do you have questions, inquiries, suggestions for improvement or for other reasons want to get in touch with us? Here you will find our contact information."
+          goto_contact: "Kontakt oss"
+      menu:
+        title: "Menu"
+        voluntary: |
+          Lyche is a restaurant and liquor bar run exclusively by volunteer students.
+          About 100 dedicated amateurs work as chefs and bar waiters and do their best for you as a guest to have a pleasant experience.
+        vision: |
+          At Lyche, we want to emphasize high-quality food and drinks.
+          This applies both to having a varied but also a well-thought-out selection.
+          We make our dishes with a focus on making them from scratch with local ingredients.
+        member_price: |
+          The menu states two prices for both drinks and food.
+          The former is a membership discount for members of the Student Society.
```

Comment:
"is a discount" -> "includes a discount"
Kan bli kjipt for KSG hvis noen misforstår denne

Reply:
Enig!